src/testRunner/unittests/tsserver/projectErrors.ts

```diff
@@ -861,4 +861,92 @@ declare module '@custom/plugin' {
             ]);
         });
     });
+
+    describe("unittests:: tsserver:: Project Errors with resolveJsonModule", () => {
+        function createSessionForTest({ include }: { include: readonly string[]; }) {
+            const test: File = {
+                path: `${projectRoot}/src/test.ts`,
+                content: `import * as blabla from "./blabla.json"`

```

Comment:
If you add a `console.log(blabla)`, you won’t have to enumerate the unrelated `_0_is_declared_but_its_value_is_never_read` diagnostic in each test, which might make the intent of the test more clear.